aten/src/ATen/native/quantized/cpu/qnnpack/include/qnnpack_func.h

```diff
@@ -0,0 +1,110 @@
+#pragma once
+#include <conv_utils.h>
+
+namespace qnnpack {
+class PrePackConvWeights {
```

Comment:
I suggest marking these two classes here final to make it clear that they are not supposed to be inherited from (which I think is the intention), otherwise the destructor should be virtual to guard against the scenario where an object of the the derived class is 'delete'd via a pointer to the base.